travel.py

```diff
@@ -9,6 +9,44 @@
 sys.setdefaultencoding("utf-8")
 
 
+# 'nick' : ['dep', 'dest', 'time', 'date']
+searched = {}
+
+
+def send_output(nick, dep, dest, date, result=None):
+    """Sends output"""
+    out = 'Nothing found'
+
+    if date == '':
+        date = datetime.datetime.today().strftime('%d.%m.%Y')
+
+    if len(result):
```

Comment:
@Adman this is abuse of notation. Please make this a boolean expression, something like `if len(result) >= 1`